Makefile

```
@@ -499,25 +499,11 @@ endef
 # Catch everything and parse the command line ourselves.
 .PHONY: %
 %:
-	# Check if we have the CMP tool installed
-	cmp $(ROOT_DIR)/Makefile $(ROOT_DIR)/Makefile >/dev/null 2>&1; if [ $$? -gt 0 ]; then printf "$(MSG_NO_CMP)"; exit 1; fi;
 	# Ensure that bin/qmk works. This will be a failing check after the next develop merge
```

```suggestion
	# Check if we have the CMP tool installed
	cmp $(ROOT_DIR)/Makefile $(ROOT_DIR)/Makefile >/dev/null 2>&1; if [ $$? -gt 0 ]; then printf "$(MSG_NO_CMP)"; exit 1; fi;
	# Ensure that bin/qmk works. This will be a failing check after the next develop merge
```

`cmp` is still used in tmk_core/rules.mk, so we unfortunately still have to check this.

---

The point here is that we dont check for any of the other dependencies, so checking *just* cmp is kinda pointless. This sort of thing is better handled in `qmk doctor`